xml/System.Diagnostics.Tracing/EventLevel.xml

```diff
@@ -211,7 +215,7 @@ The following example shows how to use the `Verbose` enumeration member to ident
       </ReturnValue>
       <MemberValue>0</MemberValue>
       <Docs>
-        <summary>No level filtering is done on the event.</summary>
+        <summary>No level filtering is done on the event. NOTE: This does not mean that you can use this level for enabling all events. Read the remarks below.</summary>
```

Comment:
```suggestion
        <summary>No level filtering is done on the event. NOTE: This level does not enable all events. For more information, see the Remarks section.</summary>
```

Reply:
@gewarren - are we free to sort these enumeration values by their order of severity in the doc? Right now it appears they are sorted alphabetically.

Reply:
You could sort them, but the next time we refresh the API docs, it will go back to alphabetical. We do have a [feature request](https://dev.azure.com/ceapex/Engineering/_workitems/edit/321397) to be able to sort the fields by value instead of name.